translate/translate.js

```diff
@@ -132,13 +153,17 @@ var program = module.exports = {
 
 cli
   .demand(1)
-  .command('detect <text>', 'Detect the language of the provided text', {}, function (options) {
-    program.detectLanguage(options.text, options.apiKey, utils.makeHandler(false));
+  .command('detect <input..>', 'Detect the language of the provided text', {}, function (options) {
```

Comment:
Can we have multiple arguments for input? (i.e. Is input a `string[]`, as opposed to a `string`?)

If so, we may want to add an "or texts" at the end of the command description.